examples/disagg_prefill/1p1d/k8s/1p1d-lws.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  name: vllm-pd-lws
+spec:
+  replicas: 1
+  leaderWorkerTemplate:
+    size: 3  # leader + 2 workers
+    restartPolicy: RecreateGroupOnPodRestart
+    leaderTemplate:
+      metadata:
+        labels:
+          role: leader
+          app: pd-proxy
+      spec:
+        containers:
+        - name: pd-proxy
+          image: lmcache/vllm-openai:latest
+          # Pod-0 is the proxy
+          command: ["/bin/bash", "-c"]
+          args:
+          - |
+            source /opt/venv/bin/activate
+            python3 /workspace/LMCache/examples/disagg_prefill/1p1d/disagg_proxy_server_first_token_from_decoder.py \
+              --host 0.0.0.0 \
+              --port 9000 \
+              --decoder-host  vllm-pd-lws-0-1.vllm-pd-lws \
+              --decoder-port 8200 \
+              --prefiller-host vllm-pd-lws-0-2.vllm-pd-lws \
+              --prefiller-port 8200
+          ports:
+          - containerPort: 9000
+    workerTemplate:
+      metadata:
+        labels:
+          role: worker
+          app: vllm-pd-instance
+      spec:
+        containers:
+        - name: vllm-pd-instance
+          image: lmcache/vllm-openai:latest
+          # Pod-0-1 is the Decoder
+          # Pod-0-2 is the Prefiller
+          command: ["/bin/bash", "-c"]
+          args:
+          - |
+            source /opt/venv/bin/activate
+            # Processing config files
+            decoder_svc="vllm-pd-lws-0-1.vllm-pd-lws" # worker-1 svc created by LWS
+            cfg_path="/workspace/LMCache/examples/disagg_prefill/1p1d/configs"
+            prefill_conf="${cfg_path}/lmcache-prefiller-config.yaml"
+            decoder_conf="${cfg_path}/lmcache-decoder-config.yaml"
+            sed -i s/localhost/${decoder_svc}/ $prefill_conf
+            sed -i s/localhost/0.0.0.0/ $decoder_conf
+            MODEL="Qwen/Qwen2.5-0.5B"
```

Comment:
MODEL via ENV would be more flexible ？

Reply:
This is just a brainstorm and doesn’t affect the current functionality.

Reply:
done, thanks